scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml

```diff
@@ -22,9 +22,11 @@ apt_packages:
   - "gcc"
```

Comment:
Hey @mingweishih , are these components needed for anything other than the code coverage components you have added?

If not, we should probably not add them here as they would be added by every user using the ansible scripts.

If we take a look [here](https://github.com/openenclave/openenclave/tree/master/scripts/ansible) this [yaml](https://github.com/openenclave/openenclave/blob/master/scripts/ansible/oe-contributors-acc-setup.yml) and [this](https://github.com/openenclave/openenclave/blob/master/scripts/ansible/oe-contributors-acc-setup-no-driver.yml) differ slightly because one doesn't install the driver. I think you want something like this whereas we have a template for constructing code coverage bits that includes all the user experience, but then also adds

```
    - import_role:
        name: linux/cicd
        tasks_from: code-coverage.yml
```

then maybe your code coverage.yml could contain something like
```

apt_packages:
  - "lcov"
  - "python3-pip"
``` 

and then if we add it [here](https://github.com/openenclave/openenclave/blob/master/scripts/ansible/oe-linux-acc-setup.yml) is probably the best way to do it.

Your previous observation that we would need to rebuild the images is, unfortunately, true. But if you check it works locally and then do something like [this](https://github.com/openenclave/openenclave/blob/master/.jenkins/pipelines/Azure/Linux/Jenkinsfile#L32) line you will be 99% sure it works and then we can rebuild everything and get you merged in.